src/components/BlockQuote.tsx

```diff
@@ -7,7 +7,7 @@ const BlockQuote = (props: BoxProps) => (
     px={4}
```

Comment:
The dark mode is not ready for production use, as some color combinations are unreadable.

That dark/light mode toggle is still for our use only, right? We're not accidentally turing it on for the public in this PR?

Reply:
The dark mode is still hidden in production. In order to get it officially released, we need to address #697.

Reply:
Okay, just checking!